src/compiler/checker.ts

```diff
@@ -2882,6 +2887,31 @@ namespace ts {
             }
         }
 
+        // Returns true if the interface given by the symbol is free of "this" references. Specifically, the result is

```

Comment:
nit: use jsdoc. I know we are not really consistent around comment format but it is very helpful to get jsdoc in the language service